ui/pages/confirmations/components/confirm-gas-display/confirm-legacy-gas-display/confirm-legacy-gas-display.js

```diff
@@ -42,14 +43,22 @@ const ConfirmLegacyGasDisplay = ({ 'data-testid': dataTestId } = {}) => {
   const unapprovedTxs = useSelector(getUnapprovedTransactions);
   const transactionData = useDraftTransactionWithTxParams();
   const txData = useSelector((state) => txDataSelector(state));
-  const { id: transactionId, dappSuggestedGasFees } = txData;
+  const { id: transactionId, dappSuggestedGasFees, layer1GasFee } = txData;
   const transaction = Object.keys(transactionData).length
     ? transactionData
     : unapprovedTxs[transactionId] || {};
   const { hexMinimumTransactionFee, hexMaximumTransactionFee } = useSelector(
     (state) => transactionFeeSelector(state, transaction),
   );
 
+  const estimatedHexMinFeeTotal = layer1GasFee
+    ? addHexes(hexMinimumTransactionFee, layer1GasFee ?? 0)
+    : hexMinimumTransactionFee;
```

Comment:
Is this ternary needed? Would it be simpler to always call addHexes but with 0x0 as the second arg if the layer 1 fee is undefined or null?

Reply:
It may work fine as is, but should the 0x0 be strings as otherwise they will evaluate to a zero number which I don't know if that function supports.

Reply:
ah. good call since the function itself specified strings w/ TS 

Interestingly, it appears the method could support other types if it wasn't for the explicit TS specification. The Numeric constructor params accept a NumericValue which is coerced to a BigNumber
`export type NumericValue = string | number | BN | BigNumber;`